packages/trip-form/i18n/es.yml

```diff
@@ -39,6 +39,9 @@ otpUi:
     use: Usar
     useCompanies: Usar las empresas
     walkOnly: Solo caminar
+  TripOptions:
+    transitOnly: Ir en transporte público
+    openApp: Abrir aplicación
```

Comment:
@miles-grant-ibigroup I found a couple of places where there was hard-coded text in `TripOptions` and hooked that into the i18n setup.  

How do you handle generating the strings in all of the languages that you support?  I've only done English and Spanish so far and used ChatGPT to generate the latter

Reply:
Just saw. Looks good! Thanks.

We pay professional translators

Reply:
I just added French, also via ChatGPT, to get the CI to pass.  Does this cover our bases with the i18n strings enough for this to be merged?

Reply:
@grant-humphries Should do for this time, however in other instances, you should check that the produced text fits in the space available, or is not disproportionally longer than the original text. (French and Spanish can produce long sentences on direct translations.)